src/components/KPrompt/KPrompt.vue

```diff
@@ -112,11 +115,15 @@ const props = defineProps({
     type: Object as PropType<ModalAttributes>,
     default: () => ({}),
   },
+  errorMessage: {
+    type: String,
+    default: 'Please enter the text as indicated above.',
```

Comment:
```suggestion
    default: 'You must enter the text to proceed.',
```

Reply:
It's not just enter the text, it also has to match.. so how about _You must enter the text as indicated above to proceed._?

Reply:
sgtm, except what about `"to confirm" instead of "proceed"?

Reply:
Done